scripts/update_data.py

```diff
@@ -0,0 +1,138 @@
+"""
+Installs the most up to date genome data on the file system that
+is required for development
+
+TODO move utility classes to other modules
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import functools
+import os
+import requests
+import sys
+import tarfile
+import time
+
+import humanize
+
+
+dataDirPath = 'ga4gh-example-data'
+zipFilePath = 'ga4gh-example-data.tar.gz'
+archivePath = 'ga4gh-example-data-archive'
+
+
+class Timed(object):
+    """
+    Decorator that times a method, reporting runtime at finish
+    """
+    def __call__(self, func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            self.start = time.time()
+            result = func(*args, **kwargs)
+            self.end = time.time()
+            self._report()
+            return result
+        return wrapper
+
+    def _report(self):
+        delta = self.end - self.start
+        timeString = humanize.time.naturaldelta(delta)
+        log("Finished in {} ({} seconds)".format(timeString, delta))
+
+
+class FileDownloader(object):
```

Comment:
We do something very similar in `process_schemas.py`. Maybe we could have a `scripts/utils.py` module and reuse the FileDownloader here and in `process_schemas`? We'll need to start worrying about where these things are run, but it should be workable.